src/composables/useRunning/useRunning.ts

```diff
@@ -15,11 +16,13 @@ export const useRunning = () => {
     const start = () => {
         isRunning.value = true;
         vigad.startTesseract();
+        startNotification();
```

Comment:
I dont think you need to create another function to generate a notification, because to generate on is already very short and descriptiv i think you can just use the function code right inside here